src/App.tsx

```diff
@@ -1,61 +1,99 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { ToDo } from './props/TODO';
+
+const orijenTodos: ToDo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(us => todo.userId === us.id),
+  };
+});
+
+export const App: React.FC = () => {
+  const [userID, setUserID] = useState(0);
+  const [nameTask, setNameTask] = useState('');
+  const [userIDError, checkUserIDError] = useState(false);
```

Comment:
use prefix 'is...' or 'has...' for boolean state variables, like 'isError' or 'hasError'